core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -690,6 +690,42 @@ public enum CoreError implements ScalarDbError {
       ""),
   DATA_LOADER_ERROR_METHOD_NULL_ARGUMENT(
       Category.USER_ERROR, "0151", "Method null argument not allowed", "", ""),
+  OUT_OF_RANGE_COLUMN_VALUE_FOR_DATE(
+      Category.USER_ERROR,
+      "0152",
+      "This Date column value is out of the valid range. It must be from 1000-01-01 through 9999-12-12. Value: %s",
```

Comment:
I tend to see column names spelled either all capitalized ([example](https://docs.aws.amazon.com/redshift/latest/dg/r_Datetime_types.html)) or all lowercase ([example](https://www.postgresql.org/docs/current/datatype-datetime.html)).

Should this capitalization stay as is? Or should we use one of the following instead?

## All capitalized

```suggestion
      "This DATE column value is out of the valid range. It must be between 1000-01-01 and 9999-12-12. Value: %s",
```

## All lowercase

```suggestion
      "This date column value is out of the valid range. It must be between 1000-01-01 and 9999-12-12. Value: %s",
```

> [!NOTE]
>
> The style we choose here should be applied to the other release notes added in this PR.

Reply:
So far in the Javadoc comment, we are using the all-capitalized naming. 
If there are no objections, I will update the error code to use all-capitalized type names.

Reply:
All-capitalized naming sounds good to me! I'll add a note about that to our style guide.

Reply:
I applied all your suggestions, thank you.